rotkehlchen/db/upgrades/v41_v42.py

```diff
@@ -136,3 +154,5 @@ def upgrade_v41_to_v42(db: 'DBHandler', progress_handler: 'DBUpgradeProgressHand
         progress_handler.new_step()
         _add_calendar_table(write_cursor)
         progress_handler.new_step()
+        _remove_manualcurrent_oracle(write_cursor)
+        progress_handler.new_step()
```

Comment:
this will increment the step by one, but total steps are set as 4 above. It should be 5.